### ISSUE TITLE
Pasting into cells of table.el breaks table

### ISSUE THREAD

**Issue by WinogradSchema** [syl20bnr/spacemacs#13821]
#### Description :octocat:
Pasting into cells of table.el

#### Reproduction guide :beetle:
- Start Emacs
- Open org buffer
- org-table-create-with-table.el (, t N)
- 4x7
- Open table editor with C-c C-'
- Paste a large chunk of text
- Try C-c C-c to adjust

*Observed behaviour:* :eyes: :broken_heart:

Table no longer adjusts its cell width. Nearby cells no longer detected. Odd
characters appearing.

*Expected behaviour:* :heart: :smile:

C-c C-c would adjust table cell on paste similar to how it does on manually
filling cells

#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.3
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 04b80d610)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(table-manipulation
 (chinese :variables chinese-enable-youdao-dict t)
 pdf lsp gtags search-engine python latex csv html java django c-c++ javascript kotlin w3m emms auto-completion better-defaults emacs-lisp git helm lsp markdown multiple-cursors
 (org :variables org-enable-github-support t org-projectile-file "TODOs.org" org-enable-org-journal-support t org-journal-dir "~/Dropbox/Org/Shared/journal/" org-journal-file-format "%Y%m%d.org" org-image-actual-width 450 org-latex-pdf-process
      '("xelatex -interaction nonstopmode %f" "xelatex -interaction nonstopmode %f")
      org-hide-emphasis-markers t org-journal-enable-agenda-integration t)
 (shell :variables shell-default-height 30 shell-default-position 'bottom)
 spell-checking syntax-checking version-control treemacs)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND DBUS GSETTINGS GLIB NOTIFY ACL LIBSELINUX GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS XWIDGETS LCMS2

#### Backtrace :paw_prints:
```
Not relevant
```

**Comment by Burkean** [comment]
What I meant was that C-c C-c would adjust the table or it would automatically happen on paste. Manually filling the cell in insert mode adjusts the cell width fine.

**Comment by duianto** [comment]
Confirmed.

The issue occurs when pasting with the evil paste commands:
In Spacemacs: `p` calls `spacemacs/evil-mc-paste-after`
but it also happens when calling: `evil-paste-after`

It works if one first switches to emacs-state by pressing: `C-z`
in the org-src buffer, and then pastes with: `C-y` which calls `*table--cell-yank`

A PR has been opened with a possible fix.
Fix org table evil paste #13845

**Comment by duianto** [comment]
The key binding in step 5 in the reproduction steps
>Open table editor with C-c C-'

seems to be a typo, because it returns:
>C-c C-' is undefined

Both `C-c '`, and `, '` works, they call `org-edit-special`

And `C-c C-c` which is mentioned in reproduction step 7:
>Try C-c C-c to adjust

in the expected behavior and in the last comment.

For me `C-c C-c` opens a sub menu with `table-` commands:

![emacs_2020-08-15_17-55-47](https://user-images.githubusercontent.com/13420573/90316172-ab0e2580-df20-11ea-9efb-d5f0095c73ca.png)

are you referring to a specific command in that list?

**Comment by Burkean** [comment]
Apologies for the lack of clarity.

`C-c ,` and `, '` are indeed the right command. What I meant by "C-c C-c would adjust the table or it would automatically happen on paste" is that my expected behaviour is that either (i) pasting in to table.el would cause the table to align itself automatically, or (ii) pasting into table.el would break the table but would be fixable by some keybinding.

It appears you understood the issue and went on to fix it correctly. Thanks!

**Comment by duianto** [comment]
The PR: Fix org table evil paste #13845
has been merged into the `develop` branch.
Feel free to close this if the issue is solved for you.
